src/components/modal-container.js

```diff
@@ -301,16 +301,11 @@ export default function ModalContainerFactory(
             };
             break;
           case EXPORT_IMAGE_ID:
-            const { ratio, legend, resolution, exporting, imageDataUri } = uiState.exportImage;
             template = (
               <ExportImageModal
+                {... uiState.exportImage}
```

Comment:
there is a space between __...__ and uiState.exportImage

Reply:
fixed